algorithms/kernel/pca/oneapi/pca_dense_correlation_online_kernel_ucapi_impl.i

```diff
@@ -0,0 +1,146 @@
+/* file: pca_dense_correlation_online_kernel_ucapi_impl.i */
+/*******************************************************************************
+* Copyright 2020 Intel Corporation
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*     http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*******************************************************************************/
+
+/*
+//++
+//  Implementation of PCA Online Kernel for GPU.
+//--
+*/
+
+#ifndef __PCA_DENSE_CORRELATION_ONLINE_KERNEL_UCAPI_IMPL__
+#define __PCA_DENSE_CORRELATION_ONLINE_KERNEL_UCAPI_IMPL__
+
+#include "externals/service_ittnotify.h"
+DAAL_ITTNOTIFY_DOMAIN(pca.dense.correlation.online.oneapi);
```

Comment:
a nip: it is better to instantiate domain directly in dispatcher.cpp file which includes this .i file, because .i may be included somewhere else (as you do in `#include "algorithms/kernel/covariance/oneapi/covariance_oneapi_impl.i"`). This is not critical.

Reply:
I did it like you in batch impl =)